hyperion/bin/preprocess_audio_files.py

```diff
@@ -69,6 +70,10 @@ def process_audio_files(
                 logging.info("Processing audio %s" % (key))
                 t2 = time.time()
 
+                if output_sampling_rate is not None:
+                    x = signal.resample(x, int(x.shape[0]*output_sampling_rate/fs))
```

Comment:
signal resample may not be a good option, I don't know if it could affect audio quality. I used this function for the VAD because I just wanted to stretch it from frame-level to sample level vad. But I don't know if this function is good for audio. Could you check the audios you got?